roachtest: sqlsmith/setup=rand-tables/setting=no-mutations failed

roachtest.sqlsmith/setup=rand-tables/setting=no-mutations [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5484799?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5484799?buildTab=artifacts#/sqlsmith/setup=rand-tables/setting=no-mutations) on master @ [47df1bc3a28f705e56ff4efc3d81c6fc90e096b5](https://github.com/cockroachdb/cockroach/commits/47df1bc3a28f705e56ff4efc3d81c6fc90e096b5):


```
test artifacts and logs in: /artifacts/sqlsmith/setup=rand-tables/setting=no-mutations/run_1
	sqlsmith.go:265,sqlsmith.go:305,test_runner.go:884: error: pq: internal error: crdb_internal.request_statement_bundle(): malformed input: expected sampling probability in range [0.0, 1.0], got 1.009579
		stmt:
		WITH
			with_2875 (col_16368, col_16369)
				AS (
					SELECT
						*
					FROM
						(
							VALUES
								(
									crdb_internal.request_statement_bundle('!sU&':::STRING::STRING, 1.0095794495824726:::FLOAT8::FLOAT8, '-13 years -9 mons -147 days -21:27:58.008122':::INTERVAL::INTERVAL, '-16 years -6 mons -129 days -20:50:08.761661':::INTERVAL::INTERVAL)::BOOL,
									'1976-07-04 20:18:06.000177':::TIMESTAMP
								),
								(false, '2025-09-24 06:20:20.000603':::TIMESTAMP),
								(COALESCE(false, NULL), '2015-12-12 16:15:51.000204':::TIMESTAMP)
						)
							AS tab_7543 (col_16368, col_16369)
				)
		SELECT
			'1988-07-14 01:28:52.000991':::TIMESTAMP AS col_16370, NULL AS col_16371
		FROM
			with_2875 AS cte_ref_831
		WHERE
			cte_ref_831.col_16368
		ORDER BY
			cte_ref_831.col_16368 ASC;
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
/cc @cockroachdb/sql-queries
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*sqlsmith/setup=rand-tables/setting=no-mutations.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-16763

roachtest.sqlsmith/setup=rand-tables/setting=no-mutations [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5500557?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5500557?buildTab=artifacts#/sqlsmith/setup=rand-tables/setting=no-mutations) on master @ [8d34ef1ea15850ee1c70470610b6652df4c317de](https://github.com/cockroachdb/cockroach/commits/8d34ef1ea15850ee1c70470610b6652df4c317de):


```
test artifacts and logs in: /artifacts/sqlsmith/setup=rand-tables/setting=no-mutations/run_1
	sqlsmith.go:265,sqlsmith.go:305,test_runner.go:897: error: pq: internal error: crdb_internal.request_statement_bundle(): malformed input: expected sampling probability in range [0.0, 1.0], got 1.550910
		stmt:
		WITH
			with_15218 (col_86735)
				AS (
					SELECT
						*
					FROM
						(
							VALUES
								(
									postgis_getbbox('01010000C0C4629D1D2EEDE3412A87ED3EE487F041741DAC31F95301C2D8C97CE944C2EBC1':::GEOMETRY::GEOMETRY)::BOX2D
								),
								('BOX(0.658128994446544 0.16437866324296135,1.0306320940638254 1.2763092075324085)':::BOX2D),
								('BOX(-0.8931558444251058 0.05731840818915185,0.6163816490717362 0.1892817567445072)':::BOX2D)
						)
							AS tab_38596 (col_86735)
				)
		SELECT
			cte_ref_4396.col_86735 AS col_86736
		FROM
			with_15218 AS cte_ref_4396
		WHERE
			crdb_internal.request_statement_bundle(e',\x7f)B':::STRING::STRING, 1.550910225586546:::FLOAT8::FLOAT8, '-1 years -1 mons -345 days -10:56:54.774168':::INTERVAL::INTERVAL, '51 years 6 mons 847 days 17:26:10.740685':::INTERVAL::INTERVAL)::BOOL
		LIMIT
			2:::INT8;
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*sqlsmith/setup=rand-tables/setting=no-mutations.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.sqlsmith/setup=rand-tables/setting=no-mutations [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5505694?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5505694?buildTab=artifacts#/sqlsmith/setup=rand-tables/setting=no-mutations) on master @ [8d34ef1ea15850ee1c70470610b6652df4c317de](https://github.com/cockroachdb/cockroach/commits/8d34ef1ea15850ee1c70470610b6652df4c317de):


```
test artifacts and logs in: /artifacts/sqlsmith/setup=rand-tables/setting=no-mutations/run_1
	sqlsmith.go:265,sqlsmith.go:305,test_runner.go:897: error: pq: internal error: crdb_internal.request_statement_bundle(): malformed input: expected sampling probability in range [0.0, 1.0], got 2.376310
		stmt:
		WITH
			with_31182 (col_153450)
				AS (
					SELECT
						*
					FROM
						(
							VALUES
								(B'0111111111111111111111111111111111111111111111'),
								(B'1101111000111011011011101100111100100110101100'),
								(B'0000111110100101101001100100111011101001011010'),
								(B'1100001010000100100010010110111100000010111000')
						)
							AS tab_81217 (col_153450)
				)
		SELECT
			B'0100000001010100110111010010110011001001111111' AS col_153451
		FROM
			with_31182 AS cte_ref_8918
		WHERE
			crdb_internal.request_statement_bundle(e'Q.\x11':::STRING::STRING, 2.376310412390721:::FLOAT8::FLOAT8, '1 mon 1 day':::INTERVAL::INTERVAL, '-49 years -6 mons -906 days -14:35:16.46972':::INTERVAL::INTERVAL)::BOOL
		ORDER BY
			cte_ref_8918.col_153450 DESC, cte_ref_8918.col_153450;
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*sqlsmith/setup=rand-tables/setting=no-mutations.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.sqlsmith/setup=rand-tables/setting=no-mutations [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5517321?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5517321?buildTab=artifacts#/sqlsmith/setup=rand-tables/setting=no-mutations) on master @ [377b113bd7e7b679c55a0f93e0d8ab5dbb6f2e97](https://github.com/cockroachdb/cockroach/commits/377b113bd7e7b679c55a0f93e0d8ab5dbb6f2e97):


```
test artifacts and logs in: /artifacts/sqlsmith/setup=rand-tables/setting=no-mutations/run_1
	sqlsmith.go:265,sqlsmith.go:305,test_runner.go:897: error: pq: internal error: crdb_internal.request_statement_bundle(): malformed input: expected sampling probability in range [0.0, 1.0], got -0.529163
		stmt:
		WITH
			with_91039 (col_447457, col_447458, col_447459)
				AS (
					SELECT
						*
					FROM
						(
							VALUES
								(1.2676927452797524:::FLOAT8, NULL, '+Inf':::FLOAT8),
								((-1.956086454537635):::FLOAT8, e'\x1d':::STRING, (-0.3081853361621475):::FLOAT8),
								(
									(1.4099166485883226:::FLOAT8::FLOAT8 % 0.44913677271236896:::FLOAT8::FLOAT8)::FLOAT8,
									NULL,
									(0.1504685891224905:::FLOAT8::FLOAT8 // (-0.4870903531550842):::FLOAT8::FLOAT8)::FLOAT8
								),
								(0.6830556279123436:::FLOAT8, 'S':::STRING, (-0.47535705801781347):::FLOAT8)
						)
							AS tab_229992 (col_447457, col_447458, col_447459)
				)
		SELECT
			((-0.591374930374416):::FLOAT8::FLOAT8 % cte_ref_26362.col_447457::FLOAT8)::FLOAT8 AS col_447460,
			cte_ref_26362.col_447458 AS col_447461,
			cte_ref_26362.col_447457 AS col_447462
		FROM
			with_91039 AS cte_ref_26362
		WHERE
			crdb_internal.request_statement_bundle('dp_5M':::STRING::STRING, (-0.5291629063743302):::FLOAT8::FLOAT8, '70 years 10 mons 23 days 10:37:31.932146':::INTERVAL::INTERVAL, '-74 years -7 mons -4 days -01:36:34.917123':::INTERVAL::INTERVAL)::BOOL
		LIMIT
			73:::INT8;
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*sqlsmith/setup=rand-tables/setting=no-mutations.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/internal/sqlsmith/schema.go
+++ b/pkg/internal/sqlsmith/schema.go
@@ -505,6 +505,7 @@ var functions = func() map[tree.FunctionClass]map[oid.Oid][]function {
 			"crdb_internal.replication_stream_progress",
 			"crdb_internal.complete_replication_stream",
 			"crdb_internal.revalidate_unique_constraint",
+			"crdb_internal.request_statement_bundle",
 		} {
 			skip = skip || strings.Contains(def.Name, substr)
 		}

